runtime/src/bank/builtins/core_bpf_migration/mod.rs

```diff
@@ -596,4 +630,54 @@ pub(crate) mod tests {
             expected_post_migration_accounts_data_size_delta_off_chain
         );
     }
+
+    #[test]
```

Comment:
Can you add a test or an assert to an existing test that demonstrates an authority change to `None` in the course of migration?

Reply:
Done!